sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -1,3 +1,189 @@
-# Azure Messaging EventHub service
+<!-- cspell:words godoc azeventhubs  -->
+# Azure Event Hubs Client Package for C++
 
-C++ SDK for Azure Messaging EventHub.
+[Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) is a big data streaming platform and event ingestion service from Microsoft. For more information about Event Hubs see: [link](https://docs.microsoft.com/azure/event-hubs/event-hubs-about).
+
+Use the client library `github.com/Azure/azure-sdk-for-cpp/sdk//eventhubs` in your application to:
+
+- Send events to an event hub.
+- Consume events from an event hub.
+
+Key links:
+- [Source code][source]
+- [API Reference Documentation][godoc]
+- [Product documentation](https://azure.microsoft.com/services/event-hubs/)
+- [Samples][godoc_examples]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Event Hubs client module for Go with `go get`:
+
+```bash
+vcpkg install azure-messaging-eventhubs-cpp
+```
+
+### Prerequisites
+
+- A C++ Compiler with C++14 support
+- An [Azure subscription](https://azure.microsoft.com/free/)
+- An [Event Hub namespace](https://docs.microsoft.com/azure/event-hubs/).
+- An Event Hub. You can create an event hub in your Event Hubs Namespace using the [Azure Portal](https://docs.microsoft.com/azure/event-hubs/event-hubs-create), or the [Azure CLI](https://docs.microsoft.com/azure/event-hubs/event-hubs-quickstart-cli).
+
+### Authenticate the client
+
+Event Hub clients are created using an Event Hub a credential from the [Azure Identity package][azure_identity_pkg], like [DefaultAzureCredential][default_azure_credential].
+You can also create a client using a connection string.
+
+<!-- NOTE: GO Links are intentional for now -->
+#### Using a service principal
+ - ConsumerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewConsumerClient)
+ - ProducerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewProducerClient)
+
+<!-- NOTE: GO Links are intentional for now -->
+#### Using a connection string
+ - ConsumerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewConsumerClientFromConnectionString)
+ - ProducerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewProducerClientFromConnectionString)
```

Comment:
Ok, looking a the note in the comment, I am assuming we have work items to add equivalent samples in C++ beyond what's being introduced in this PR, and these are just placeholders?

I recommend removing them before the beta release to avoid confusion (in case we don't have samples until then).

Reply:
Those samples exist today. But they have not been pushed to the SDK documentation site. But I can't have dead links here, so I have to link to *something*.